### PR TITLE
Recommit on top of last IB the recipe to run the photon validation on…

### PR DESCRIPTION
**Pull request by nancymarinelli**
… phase 2 RelVals

#### PR description:

Recipe to run the photon validation in Phase2 

#### PR validation:

Succesfully tested with 
runTheMatrix.py --what upgrade -b 'UPSG_Std_2026D41noPU' -l 20452.0 --label 2026D41noPU -m 5000 -t 8 --noCafVeto --command="--customise SLHCUpgradeSimulations/Configuration/aging.customise_aging_1000 -n 100"  >& localRun.log &

#### if this PR is a backport please specify the original PR:

Before submitting your pull requests, make sure you followed this checklist:
- verify that the PR is really intended for the chosen branch
- verify that changes follow [CMS Naming, Coding, And Style Rules](http://cms-sw.github.io/cms_coding_rules.html)
- verify that the PR passes the basic test procedure suggested in the [CMSSW PR instructions](https://cms-sw.github.io/PRWorkflow.html)

### CODE REVIEW

**Comment by jfernan2** [comment]
please test workflow 20452.0,20652.0,20652.98,20652.99

**Comment by jfernan2** [comment]
please test
Not sure how to enable those workflows belonging to --what upgrade here in the bot, any advice?

File: Validation/Configuration/python/autoValidation.py [hunk 1]
```diff
@@ -16,10 +17,9 @@
                    'MTDValidation' : ['', 'globalValidationMTD', 'mtdValidationPostProcessor'],
                    'OuterTrackerValidation' : ['', 'globalValidationOuterTracker', 'postValidationOuterTracker'],
                    'ecalValidation_phase2' : ['', 'validationECALPhase2', ''],
-                   'TrackerPhase2Validation' : ['', 'trackerphase2ValidationSource', 'trackerphase2ValidationHarvesting'],
                  }
 
-_phase2_allowed = ['baseValidation','trackingValidation','muonOnlyValidation','JetMETOnlyValidation', 'electronOnlyValidation', 'bTagOnlyValidation', 'tauOnlyValidation', 'hcalOnlyValidation', 'HGCalValidation', 'MTDValidation', 'OuterTrackerValidation', 'ecalValidation_phase2', 'TrackerPhase2Validation']
```
Comment (jfernan2) [inline Validation/Configuration/python/autoValidation.py]: You dropped by mistake TrackerPhaseValidation, please put it back. Thanks

**Comment by jfernan2** [comment]
-1

**Comment by nancymarinelli** [comment]
Hello
I will check, but I used the latest IB to submit this PR


On 29/11/19 21:56, jfernan2 wrote:
> jfernan2 commented on this pull request.
>
>
>
>>                    }
>   
> -_phase2_allowed = ['baseValidation','trackingValidation','muonOnlyValidation','JetMETOnlyValidation', 'electronOnlyValidation', 'bTagOnlyValidation', 'tauOnlyValidation', 'hcalOnlyValidation', 'HGCalValidation', 'MTDValidation', 'OuterTrackerValidation', 'ecalValidation_phase2', 'TrackerPhase2Validation']
>
> You dropped by mistake TrackerPhaseValidation, please put it back. Thanks
>

-- 
________________________________________
Nancy Marinelli
Research Associate Professor
University of Notre Dame, IN, US
CERN, Bdg 40/3-A01, 1211 Geneva
SWITZERLAND
Phone +41-22-76-70809
fax +41-22-76-78940

File: Validation/Configuration/python/autoValidation.py (hunk 1 above)
Comment (nancymarinelli) [reply Validation/Configuration/python/autoValidation.py]: I just fixed it. Sorry about that

**Comment by jfernan2** [comment]
please test

**Comment by jfernan2** [comment]
+1

**Comment by civanch** [comment]
+1

**Comment by fabiocos** [comment]
@jfernan2 we keep a small subset of phase2 workflow in the regular list exactly for this kind of test

**Comment by fabiocos** [comment]
+1

**Comment by nancymarinelli** [comment]
Has this PR been included in 11_0_0_pre13_phase2 ?  If yes, it still does no work and I need to do more

**Comment by boudoul** [comment]
Hi @nancymarinelli , there is no specific phase2 release, each release is common for run1,run2, phase2 .

For 11_0_0_pre13 : it has been announced 10 days ago (Nov 29 )  :  https://hypernews.cern.ch/HyperNews/CMS/get/relAnnounce/2091.html, so liukely built the same days you submitted your PR, then your PRs has been merged 5 days ago, so your PR cannot be in .

FYI you can always x-check what entered in a release looking at the release notes produced for each release , example for cmssw_11_0_0_pre13 : 
https://github.com/cms-sw/cmssw/releases/CMSSW_11_0_0_pre13

Hope that helps

**Comment by jfernan2** [comment]
The Milestone for this PR was 11_1_X, hence I am afraid 11_0_0_pre13 does not contain it

**Comment by fabiocos** [comment]
@nancymarinelli this PR has been merged into CMSSW_11_1_X_2019-12-04-1100 as can be seen in https://cmssdt.cern.ch/SDT/html/cmssdt-ib/#/ib/CMSSW_11_1_X . It was not signalled as crucial for 11_0_X (whose main purpose if HLT TDR production up to RAW).

**Pull request merged**